### PR TITLE
ridingqwerty: userspace update

### PR DESCRIPTION
**Pull request by ridingqwerty**
## Description

⌨️

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/atreus/keymaps/ridingqwerty/rules.mk [hunk 1]
```diff
@@ -5,13 +5,14 @@ MCU = STM32F303
...
+MOUSEKEY_ENABLE = no # Mouse keys
 EXTRAKEY_ENABLE = yes # Audio control and System control
-CONSOLE_ENABLE = yes  # Console for debug
-COMMAND_ENABLE = yes    # Commands for debug and configuration
-#SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
+CONSOLE_ENABLE = no  # Console for debug
+COMMAND_ENABLE = no    # Commands for debug and configuration
 NKRO_ENABLE = yes     # USB Nkey Rollover
-AUDIO_ENABLE = yes
+AUDIO_ENABLE = yes   # Doot
```
Comment (Duckle29) [inline keyboards/atreus/keymaps/ridingqwerty/rules.mk]: doot

Comment (ridingqwerty) [reply keyboards/atreus/keymaps/ridingqwerty/rules.mk]: 🎺

File: users/ridingqwerty/process_records.c [hunk 2]
```diff
@@ -10,23 +13,477 @@ bool process_record_secrets(uint16_t keycode, keyrecord_t *record) {
...
+          tap_code(KC_SPC);
+        }
+        return false;
+#else
+      case A:
+      case E:
+        if (record->event.pressed) {
+          register_code(KC_LSFT);
+          tap_code16(temp_keycode);
+          unregister_code(KC_LSFT);
```
Comment (drashna) [inline users/ridingqwerty/process_records.c]: You could just do 
```suggestion
          tap_code16(LSFT(temp_keycode));
```

Comment (ridingqwerty) [reply users/ridingqwerty/process_records.c]: I’ve tested this already and believe it or not, things get screwy.

Comment (ridingqwerty) [reply users/ridingqwerty/process_records.c]: For giggles I amended it as you suggested and it works just fine now; I have no idea what had me addled previously.

**Pull request merged**